src/CodeGen_C.cpp

```diff
@@ -108,6 +108,12 @@ inline float float_from_bits(uint32_t bits) {return reinterpret<float, uint32_t>
 template<typename T> T max(T a, T b) {if (a > b) return a; return b;}
 template<typename T> T min(T a, T b) {if (a < b) return a; return b;}
 
+template<typename A, typename B>
```

Comment:
and I guess we don't want to be requiring c++11 in consumers of our generated C++, so no universal references...